client/verta/verta/_cli/registry/create.py

```diff
@@ -28,6 +30,14 @@ def create_model(model_name, label, visibility, workspace):
     """Create a new registeredmodel entry.
     """
     public_within_org = visibility == "org"
+    client = Client()
+
+    try:
+        model = client.get_or_create_registered_model(model_name, workspace=workspace, public_within_org=public_within_org)
+        for l in label:
+            model.add_label(l)
+    except ValueError:
+        raise click.BadParameter("model {} not found".format(model_name))
```

Comment:
```suggestion
    model = client.get_or_create_registered_model(model_name, workspace=workspace, public_within_org=public_within_org)
    for l in label:
        model.add_label(l)
```
No need for the `try-except` here, since they are creating a _new_ model.